source/internal.d.ts

```diff
@@ -47,6 +47,8 @@ export type WordSeparators = '-' | '_' | ' ';
 
 export type StringDigit = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9';
 
+type Whitespace = ' ' | '\t' | '\n' | '\r' | '\f' | '\v';
```

Comment:
There are a lot more whitespace characters in Unicode.

Reply:
I added a few more using this page as a reference: https://en.wikipedia.org/wiki/Whitespace_character